### PR TITLE
Development

### PR DESCRIPTION
**Pull request by priyagupta314-work**

### CODE REVIEW

File: avatar.html [hunk 1]
```diff
@@ -1,96 +1,136 @@
...
+                style="border:0; transform: scale(1); overflow:hidden;"
```
Comment (poojasabnani) [inline avatar.html]: inline style can be avoided

File: badge.html [hunk 2]
```diff
@@ -44,22 +49,26 @@ <h1 class="lib-header"> uiStore </h1>
...
-sandbox="allow-scripts allow-same-origin">
-</iframe>
+            
         </div> <!--end of class badges-->
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=blackboard&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cspan%2520class%253D%2522badge%2520badge-primary%2522%253E%2520primary%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-secondary%2522%253E%2520secondary%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-success%2522%253E%2520success%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-failure%2522%253E%2520failure%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-warning%2522%253E%2520warning%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-info%2522%253E%2520info%2520%253C%252Fspan%253E%250A%253Cspan%2520class%253D%2522badge%2520badge-special%2522%253E%2520popular%2520%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
```
Comment (poojasabnani) [inline badge.html]: this can be formatted

File: badge.html [hunk 3]
```diff
@@ -44,22 +49,26 @@ <h1 class="lib-header"> uiStore </h1>
...
+  style="border:0; transform: scale(1); overflow:hidden;"
```
Comment (poojasabnani) [inline badge.html]: in iframes the inline style which is getting repeated can be avoided by adding them to a class

File: getting-started.html [hunk 4]
```diff
@@ -5,45 +5,95 @@
...
+        <div class="main">
+            <div class="get-started">
+                <h2 id="get-started-head" class="section-head">
+                    Get started
+                </h2>
+                <h1> CSS </h1>
+                <div>
+                    <p class="component-description"> Copy the code below and add it to the &lthead&gt element of your
+                        HTML file to get started. </p>
+
```
Comment (poojasabnani) [inline getting-started.html]: extra unnecessary spaces can be removed

**Review by poojasabnani — COMMENTED** [review]
Hey @priyagupta314-work ,

Well written code. I've just suggested basic refactoring for code clean up.

**Comment by ritikarp** [comment]
Hi @priyagupta314-work 
You can make separate folders for all CSS components and images for better accessibility of code.

File: getting-started.html (hunk 4 above)
Comment (priyagupta314-work) [reply getting-started.html]: Done!

File: badge.html (hunk 3 above)
Comment (priyagupta314-work) [reply badge.html]: Done

File: badge.html (hunk 2 above)
Comment (priyagupta314-work) [reply badge.html]: done

File: avatar.html (hunk 1 above)
Comment (priyagupta314-work) [reply avatar.html]: done

**Comment by Rupamdas832** [comment]
Make cursor:  "pointer", and some changes in style ( either shadow or border) on hover over buttons. 
Give some padding to badges. 
Social links can be opened in new tabs.

**Review by Rupamdas832 — CHANGES_REQUESTED** [review]
Make cursor: "pointer", and some changes in style ( either shadow or border) on hover over buttons.
Give some padding to badges.
Social links can be opened in new tabs.

**Pull request merged**